### PR TITLE
refactor(plans): internal restructure

### PR DESCRIPTION
**Pull request by neurosnap**
Screenshot 2023-10-02 at 8 56 36 PM
Screenshot 2023-10-02 at 8 56 15 PM
Screenshot 2023-10-02 at 8 56 07 PM

### CODE REVIEW

**Review by joeclayallday — APPROVED** [review]

**Review by joeclayallday — APPROVED** [review]

**Comment by joeclayallday** [comment]
Looks great! Nice work 👏

**Review by joeclayallday — APPROVED** [review]

File: src/ui/shared/plan.tsx [hunk 1]
```diff
@@ -305,7 +306,7 @@ const PlanCard = ({
     >
       <div className="mb-8 mx-4" style={{ height: 135, minWidth: 225 }}>
```
Comment (joeclayallday) [inline src/ui/shared/plan.tsx:307]: @neurosnap If you change mx-4 to mx-auto the words fit better inside the plan divs

BEFORE
Screenshot 2023-10-02 at 8 24 39 PM

AFTER
Screenshot 2023-10-02 at 8 24 51 PM

**Comment by joeclayallday** [comment]
Updated plans descriptions:
Starter: `Deploy your app and get to market fast`
Growth: `Gain user traction and deliver more functionality`
Scale: `Run mission-critical apps at scale without worry`
Enterprise: `Meet any requirement across performance, reliability, and security`

File: src/ui/shared/plan.tsx [hunk 2]
```diff
@@ -305,7 +306,7 @@ const PlanCard = ({
...
         <div style={{ height: 95 }}>
-          <h2 className={`text-center ${tokens.type.h2}`}>
+          <h2 className={`text-center ${tokens.type.h2} ${fontColor}`}>
             {capitalize(plan.name)}
           </h2>
           <p className={"text-center mt-4 text-sm"}>
```
Comment (joeclayallday) [inline src/ui/shared/plan.tsx:312]: Add these classes `mx-auto max-w-[30ch]` to get better line breaks

Screenshot 2023-10-02 at 8 34 40 PM

**Comment by joeclayallday** [comment]
Plan descriptions are responsive and updated to match the marketing site now 👍 

Screenshot 2023-10-02 at 8 45 15 PM
Screenshot 2023-10-02 at 8 45 10 PM

**Review by joeclayallday — APPROVED** [review]

**Review by grrowe — APPROVED** [review]

**Pull request merged**